README.rst

```diff
@@ -55,7 +55,7 @@ Installation
 Dependencies
 ~~~~~~~~~~~~
 
-scikit-learn requires:
+scikit-learn prerequires:
```

Comment:
I personally prefer "require", as it echoes to `requirements.txt`, `setup(install_requires=...)`, ...